listStyle.css

```diff
@@ -0,0 +1,25 @@
+.chenar{
+    align-content: center;
```

Comment:
`align-content` si `align-self` se folosesc in sisteme flex sau grid, hai sa revedem aici

Reply:
Done!